crm_lead_to_task/models/crm_lead.py

```diff
@@ -79,16 +64,36 @@ def _action_create_and_open_task(self, project):
             "target": "current",
         }
 
-    def action_crm_to_task(self):
+    def action_create_task(self):
+        """Open wizard to create task from lead"""
         self.ensure_one()
+        action = self.env.ref("crm_lead_to_task.crm_lead_convert2task_action").read()[0]
+
+        context = {"default_lead_id": self.id}
         if self.company_id.crm_force_project_id:
-            action = self._action_create_and_open_task(
-                self.company_id.crm_force_project_id
-            )
-        else:
-            action = self.env.ref(
-                "crm_lead_to_task.crm_lead_convert2task_action"
-            ).read()[0]
-            action["context"] = {"default_lead_id": self.id}
+            context["default_project_id"] = self.company_id.crm_force_project_id.id
 
+        action["context"] = context
         return action
+
+    def _move_chatter(self, task):
```

Comment:
issue: remember that many modules can coexist, so it's better if method names are more explicit:


```suggestion
    def _move_chatter_to_task(self, task):
```